app/controllers/homes_controller.rb

```diff
@@ -3,7 +3,9 @@ def show
     if signed_out?
       render "pages/landing"
     else
-      @recommended_items = RecommendedItem.where(user: current_user.with_following)
+      @recommended_items = RecommendedItem.
+        where(user: current_user.with_following).
```

Comment:
Agree, the [rubocop config][1] needs to change in this repo. Who enjoys that much indenting?

[1]: https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L639-L646